<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsTreeDialog</class>
 <widget class="QDialog" name="StatsTreeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="statsTreeWidget">
     <column>
      <property name="text">
       <string notr="true">Item</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Display filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DisplayFilterEdit" name="displayFilterLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="applyFilterButton">
       <property name="toolTip">
        <string>Regenerate statistics using this display filter</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionCopyToClipboard">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy a text representation of the tree to the clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save the stats_tree data in various formats</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayFilterEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">display_filter_edit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StatsTreeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StatsTreeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
